android/engine/src/test/java/org/smartregister/fhircore/engine/util/DateUtilsTest.kt

```diff
@@ -38,6 +39,27 @@ class DateUtilsTest {
     Assert.assertEquals("3-22-2020", finalDate)
   }
 
+  @Test
+  fun `addDays() should return same date when given initialDate and pattern`() {
+    val finalDate =
+      DateUtils.addDays("2020-03-10 01:23:00 AM", dateTimeFormat = "yyyy-MM-dd h:mm:ss a")
+    Assert.assertEquals("3-10-2020", finalDate)
+  }
+
+  @Test
+  fun `hasPastDays should return true when given initialDate and days`() {
+    val initialDate = DateTimeType("2022-01-11T15:30:10.222")
+    val hasPastDays = DateUtils.hasPastDays(initialDate, -3)
+    Assert.assertTrue(hasPastDays)
+  }
+
+  @Test
+  fun `hasPastDays should return true when given initialDate`() {
+    val initialDate = DateTimeType("2022-01-11T15:30:10.222")
+    val hasPastDays = DateUtils.hasPastDays(initialDate)
+    Assert.assertTrue(hasPastDays)
+  }
+
   @Test
   fun `String#getDate() should return correct date when given string format expected`() {
     Assert.assertEquals("2020-03-10".getDate("yyyy-MM-dd").makeItReadable(), "10-Mar-2020")
```

Comment:
The **Expected** value should always be the first parameter. Please switch these around

Reply:
Okay, thanks, I missed that one